config/master.d/50-returner.conf

```diff
@@ -9,7 +9,8 @@ mysql:
 
 ext_pillar:
   - mysql:
-    # This queries allows us to override our pillar values up to a depth of 3.
+    # This queries allows us to override our pillar values up to a depth of 5
```

Comment:
Should be either "These queries allow us to ..." or "This query allows us to..."